packages/store-ui/src/FloatingActionButton/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react'
+import { Flex } from 'theme-ui'
+
+export interface FloatingActionButtonProps {
+  variant?: string
+  href?: string
+}
+
+export const FloatingActionButton: FC<FloatingActionButtonProps> = ({
```

Comment:
I have a feeling that default exporting is better because it allows people lazy import this component if they want